policykit/policyengine/models.py

```diff
@@ -775,8 +780,12 @@ def save(self, *args, **kwargs):
         logger.info('entered save')
         if not self.pk:
             logger.info('is pk')
+            if self.data is None:
+                ds = DataStore.objects.create()
+                self.data = ds
```

Comment:
Is this the right place to initialize this @rickygv99? I was looking at how the `data` datastore is created for Proposals [here](https://github.com/amyxzhang/policykit/blob/c6bcb88240d1bd79e8b01743222ab2fd96883dac/policykit/policyengine/models.py#L242-L244).